_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -75,6 +75,9 @@ class HealthCareApp extends React.Component {
     const path = this.props.location.pathname;
     this.context.store.dispatch(updateSubmissionStatus('applicationSubmitted'));
     this.context.store.dispatch(updateCompletionStatus(path));
+    if (document.getElementsByTagName('h4').length > 0) {
+      document.getElementsByTagName('h4')[0].scrollIntoView();
```

Comment:
:-/  Hmm... all these scrollIntoViews feel like potential race conditions.  Somethign to flag for understanding correctly later.